src/test/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKAnnotationPluginDescriptorUnitTest.java

```diff
@@ -0,0 +1,326 @@
+        Assert.assertTrue(vae.getGenotypeAnnotations().isEmpty());
+        Assert.assertTrue(vae.getInfoAnnotations().isEmpty());
+    }
+
+    @Test
+    public void testExclude(){
+        CommandLineParser clp = new CommandLineArgumentParser(
+                new Object(),
+                Collections.singletonList(new GATKAnnotationPluginDescriptor(Collections.singletonList(new Coverage()), null)),
+                Collections.emptySet());
+        String[] args = {"-AX", Coverage.class.getSimpleName()};
+        clp.parseArguments(nullMessageStream, args);
+        List<Annotation> annots = instantiateAnnotations(clp);
+        final VariantAnnotatorEngine vae = new VariantAnnotatorEngine(annots, null, Collections.emptyList(), false);
+        Assert.assertTrue(vae.getGenotypeAnnotations().isEmpty());
+        Assert.assertTrue(vae.getInfoAnnotations().isEmpty());
+    }
+
+    @Test
+    public void testIncludeDefaultGroupExcludeIndividual(){
+        CommandLineParser clp = new CommandLineArgumentParser(
+                new Object(),
+                Collections.singletonList(new GATKAnnotationPluginDescriptor(null, Collections.singletonList(StandardAnnotation.class))),
+                Collections.emptySet());
+        String[] args = {"--annotationsToExclude", Coverage.class.getSimpleName()};
+        clp.parseArguments(nullMessageStream, args);
+        List<Annotation> annots = instantiateAnnotations(clp);
+        final VariantAnnotatorEngine vae = new VariantAnnotatorEngine(annots, null, Collections.emptyList(), false);
+
+        Assert.assertFalse(vae.getGenotypeAnnotations().isEmpty());
+        Assert.assertFalse(vae.getInfoAnnotations().isEmpty());
+        //check that Coverage is out
+        Assert.assertTrue(vae.getInfoAnnotations().stream().noneMatch(a -> a.getClass().getSimpleName().equals(Coverage.class.getSimpleName())));
+    }
+
+    @Test
+    public void testIncludeUserGroupExcludeIndividual(){
+        CommandLineParser clp = new CommandLineArgumentParser(
+                new Object(),
+                Collections.singletonList(new GATKAnnotationPluginDescriptor(null, null)),
+                Collections.emptySet());
+        String[] args = {"--annotationsToExclude", Coverage.class.getSimpleName(), "-G", StandardAnnotation.class.getSimpleName()};
+        clp.parseArguments(nullMessageStream, args);
+        List<Annotation> annots = instantiateAnnotations(clp);
+        final VariantAnnotatorEngine vae = new VariantAnnotatorEngine(annots, null, Collections.emptyList(), false);
+
+        Assert.assertFalse(vae.getGenotypeAnnotations().isEmpty());
+        Assert.assertFalse(vae.getInfoAnnotations().isEmpty());
+        //check that Coverage is out
+        Assert.assertTrue(vae.getInfoAnnotations().stream().noneMatch(a -> a.getClass().getSimpleName().equals(Coverage.class.getSimpleName())));
+    }
+
```

Comment:
I think we also need testIncludeDefaultGroupExcludeGroup.

Reply:
We don't have group exclusion currently, so we won't have to worry about this test